RELEASE/scripts/autoscend/combat/auto_combat_zombie_slayer.ash

```diff
@@ -53,10 +60,148 @@ string auto_combatZombieSlayerStage3(int round, monster enemy, string text)
 
 string auto_combatZombieSlayerStage4(int round, monster enemy, string text)
 {
+	// stage 4 = prekill. copy, sing along, flyer and other things that need to be done after delevel but before killing
 	if (!in_zombieSlayer())
 	{
 		return "";
 	}
+	
+	// Basically stolen from Ed's Lash targets
+	if(canUse($skill[Smash & Graaagh]) && get_property("_zombieSmashPocketsUsed").to_int() < 30 && canSurvive(2.0)) 
```

Comment:
maybe this is a good chance to move the lash list into the monsters data file

Reply:
I have very little idea how the monster data files work, will see if its something I can manage, but cannot promise anything

Reply:
don't worry about it atm then